docs/publishers/config.rst

```diff
@@ -167,6 +187,22 @@ loads.
 
      This setting can only be set using :js:func:`window.hypothesisConfig`.
 
+   .. option:: onProfileRequest
+
+     ``function``. A JavaScript function that will be called when the user clicks
+     the user profile (user name) link in the sidebar. No arguments are passed
```

Comment:
Makes sense. Perhaps something like:

```
onProfileRequested(data)
```

Where `data` has the format:

```ts
{
  username: string;
  isCurrentUser: boolean;
}
```

Other `onXYZ` callbacks would have a similar format if there were any relevant parameters.